CentOS : Setup fails on RDPY building 'rle' extension

## OS Used -
CentOS Linux release 7.6.1810 (Core)
Intel(R) Atom(TM) CPU  C2750  @ 2.40GHz
MemTotal:        7995416 kB

## Error
The EyeWitness setup includes a section where it installs RDPY
```
    echo '[*] Installing RDPY'
    git clone https://github.com/ChrisTruncer/rdpy.git
    cd rdpy
    python setup.py install
    cd ..
    rm -rf rdpy
```

This fails on CentOS

```
running build_ext
building 'rle' extension
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/ext
gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/python2.7 -c ext/rle.c -o build/temp.linux-x86_64-2.7/ext/rle.o
ext/rle.c:27:20: fatal error: Python.h: No such file or directory
 #include <Python.h>
                    ^
compilation terminated.
error: command 'gcc' failed with exit status 1
```

## Expected behavior (vs. what you encountered)
setup should complete successfully.

## Solution
Upon investigation it transpires that the python-devel package is required. This is not included in the CentOS section of setup.py.

Upon performing a yum install python-devel and rerunning setup.py the installation of the rle extension was successful.

```
running build_ext
building 'rle' extension
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/ext
gcc -pthread -fno-strict-aliasing -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/python2.7 -c ext/rle.c -o build/temp.linux-x86_64-2.7/ext/rle.o
gcc -pthread -shared -Wl,-z,relro build/temp.linux-x86_64-2.7/ext/rle.o -L/usr/lib64 -lpython2.7 -o build/lib.linux-x86_64-2.7/rle.so
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/egg
```

--- a/setup/setup.sh
+++ b/setup/setup.sh
@@ -237,7 +237,7 @@ case ${osinfo} in
     fi
     echo '[*] Installing CentOS Dependencies'
     yum install cmake python python-pip PyQt4 PyQt4-webkit \
-                python-argparse xvfb python-netaddr python-dev tesseract-ocr firefox-esr python-devel
+                python-argparse xvfb python-netaddr python-devel tesseract-ocr firefox-esr
     echo
     echo '[*] Installing RDPY'
     git clone https://github.com/ChrisTruncer/rdpy.git

--- a/setup/setup.sh
+++ b/setup/setup.sh
@@ -237,7 +237,7 @@ case ${osinfo} in
     fi
     echo '[*] Installing CentOS Dependencies'
     yum install cmake python python-pip PyQt4 PyQt4-webkit \
-                python-argparse xvfb python-netaddr python-dev tesseract-ocr firefox-esr
+                python-argparse xvfb python-netaddr python-dev tesseract-ocr firefox-esr python-devel
     echo
     echo '[*] Installing RDPY'
     git clone https://github.com/ChrisTruncer/rdpy.git

--- a/setup/setup.sh
+++ b/setup/setup.sh
@@ -237,7 +237,7 @@ case ${osinfo} in
     fi
     echo '[*] Installing CentOS Dependencies'
     yum install cmake python python-pip PyQt4 PyQt4-webkit \
-                python-argparse xvfb python-netaddr python-devel tesseract-ocr firefox-esr
+                python-argparse xorg-x11-server-Xvfb python-netaddr python-devel tesseract firefox
     echo
     echo '[*] Installing RDPY'
     git clone https://github.com/ChrisTruncer/rdpy.git

